_posts/2019-05-28-bazel-0.26.0.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: posts
+title: "Bazel 0.26"
+authors:
+  - aehlig
+---
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.26](https://github.com/bazelbuild/bazel/releases/tag/0.26.0).
+
+## Incompatible changes
+
+- [`--incompatible_windows_escape_jvm_flags`](https://github.com/bazelbuild/bazel/issues/7486)
+  is enabled by default, and removed.
+
+- [`--incompatible_windows_style_arg_escaping`](https://github.com/bazelbuild/bazel/issues/7454)
+  is enabled by default.
+
+- [`--incompatible_no_output_attr_default`](https://github.com/bazelbuild/bazel/issues/7950)
+  is enabled by default.
+
+- [`--incompatible_depset_union`](https://github.com/bazelbuild/bazel/issues/5817)
+  is enabled by default.
+
+- [`--incompatible_disable_genrule_cc_toolchain_dependency`](https://github.com/bazelbuild/bazel/issues/6867) removed
+
+- [`--Incompatible_disallow_rule_execution_platform_constraints_allowed`](https://github.com/bazelbuild/bazel/issues/8136):
+  When enabled, disallows the use of the `execution_platform_constraints_allowed`
+  attribute when defining new rules.
+
+- [`--incompatible_disable_objc_provider_resources`](https://github.com/bazelbuild/bazel/issues/5710)
+  is now enabled by default.
+
+## Apple
+
+- The `--ios_multi_cpus`, `--watchos_cpus`, `--macos_cpus` and `--tvos_cpus` are
+  now additive. This means that you can now split the `--ios_multi_cpus=arm64,armv7`
+  into `--ios_multi_cpus=arm64` and `--ios_multi_cpus=armv7`.
+
+- Add `--incompatible_objc_framework_cleanup` to control whether to enable some
+  objc framework cleanup that changes the API. Specifically, the cleanup changes
+  the objc provider API pertaining to frameworks. This change is expected to be
+  transparent to most users unless they write their own Starlark rules to handle
+  frameworks. See https://github.com/bazelbuild/bazel/issues/7594 for details.
+
+- `objc_library` does not support resource attributes any more. Please read
+  [#7594](https://github.com/bazelbuild/bazel/issues/7594) for more info.
+
+## Android
+
+- Android resource conflicts will no longer be reported between a strong attr
+  resource and a weak attr resource, if the weak attr does not have format
+  specified.
+
+## C++
+
+- [Allow debugging C++ features logic](https://github.com/bazelbuild/bazel/commit/53156db1e30fe7197889c3cbc8d1321ff9b85143
+
+- Add new options [`--cs_fdo_instrument`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--cs_fdo_instrument)
+  and [`--cs_fdo_profile`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--cs_fdo_profile)
+  to support LLVM's context-sensitive FDO (CSFDO).
+
+- [Bazel C++ compile/link Starlark API](https://github.com/bazelbuild/bazel/issues/4570).
+  Can be used with experimental flag `--experimental_cc_skylark_api_enabled_packages`.
+
+- `cc_toolchain.static_runtime_lib` and `cc_toolchain.dynamic_runtime_lib` are
```

Comment:
These should be links

Reply:
Links to what? I couldn't find any documentation for those.

Reply:
cc @hlopko

Reply:
added links. Thank @hlopko for looking them up.